source/common/json/config_schemas.cc

```diff
@@ -211,7 +211,8 @@ const std::string Json::Schema::HTTP_CONN_NETWORK_FILTER_SCHEMA(R"EOF(
       "tracing" : {
         "type" : "object",
         "properties" : {
-          "operation_name" : {"type" : "string"}
+          "operation_name" : {"type" : "string"},
+          "enum": ["ingress", "egress"]
```

Comment:
This doesn't look correct to me. The enum should be inside the type specification on the previous line I believe. I'm not sure what this is doing exactly.